src/display/api.js

```diff
@@ -80,7 +81,7 @@ let createPDFNetworkStream;
  * data transport.
  * @param {IPDFStreamFactory} pdfNetworkStreamFactory - The factory function
  *   that takes document initialization parameters (including a "url") and
- *   returns an instance of {IPDFStream}.
+ *   returns a promise which is resolved with an instance of {IPDFStream}.
```

Comment:
Isn't this patch `[api-minor]` since the `setPDFNetworkStreamFactory` is exposed in the public API, or will old-style `pdfNetworkStreamFactory` parameters still work?

Reply:
> Isn't this patch `[api-minor]` since the `setPDFNetworkStreamFactory` is exposed in the public API,

It's not actually part of the public API though, given that it's not being exported by the https://github.com/mozilla/pdf.js/blob/master/src/pdf.js file.

> or will old-style `pdfNetworkStreamFactory` parameters still work?

Any existing code should "just" work, note also that I didn't re-write any of the existing call-sites in `src/pdf.js` either.

Reply:
Whoops, I accidentally looked at the exports of `api.js`... Yes, this can be ignored then.